CHANGELOG.md

```diff
@@ -56,6 +56,7 @@
 <a name="breaking_changes_1.16.0">[Breaking Changes:](#breaking_changes_1.16.0)</a>
 
 - [callhierarchy] `CurrentEditorAccess` is deprecated. Use the version implemented in the `editor` package instead. The services in `call-hierarchy` that previously used the local `CurrentEditorAccess` no longer do [#9681](https://github.com/eclipse-theia/theia/pull/9681)
+- [core] moved from ES5 to ES2017 [#9436](https://github.com/eclipse-theia/theia/pull/9436) - Contributed on behalf of STMicroelectronics
```

Comment:
I think this should be placed under 1.17.0 breaking changes now.

Reply:
Now that we have a migration guide, how about adding a section on what adopters will have to do?

Reply:
@tsmaeder @sgraband the migration guide is not yet merged, we can update it after if necessary:
- https://github.com/eclipse-theia/theia/pull/9817

Reply:
@sgraband you forgot to remove the old entry under 1.16.0

Reply:
I removed the old changelog entry.